launch_ros/launch_ros/actions/node.py

```diff
@@ -365,7 +367,16 @@ def execute(self, context: LaunchContext) -> Optional[List[Action]]:
                     '{}:={}'.format(remapping_from, remapping_to)
                 )
         context.extend_locals({'ros_specific_arguments': ros_specific_arguments})
-        return super().execute(context)
+        ret = super().execute(context)
+
+        self.__logger = launch.logging.get_logger(self.name)
```

Comment:
maybe leave a comment here why you're overriding the existing __logger attribute here? I think it's because we didn't have a fully qualified name until `execute` but i'm not positive

Reply:
I decided to not override the existing `__logger` attribute.